From 8bf783e4a473fdd21efe7ba6a60c9b42a8d1bcb2 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 26 Nov 2021 16:56:37 +0000
Subject: [PATCH] dtoverlays: Use edt-ft5506 for 10 points, instead of
 edt-ft5x06

Whilst all the datasheets describe FT5x06 as supporting "up to
10 points of absolution X and Y coordinates", the driver
implementation for the compatible string "edt,edt-ft5x06" only
allows for 5.

Switch to the "edt,edt-ft5506" compatible string which allows for
10 points with no other differences.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/edt-ft5406.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/edt-ft5406.dtsi
+++ b/arch/arm/boot/dts/overlays/edt-ft5406.dtsi
@@ -28,7 +28,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			ft5406: ts@38 {
-				compatible = "edt,edt-ft5406";
+				compatible = "edt,edt-ft5506";
 				reg = <0x38>;
 
 				touchscreen-size-x = < 800 >;
